vast/vast.cpp

```diff
@@ -37,89 +37,110 @@
 #include <string>
 #include <vector>
 
-using namespace vast;
-using namespace vast::system;
+namespace vast {
+
+caf::expected<std::vector<std::filesystem::path>>
+load_plugins(system::configuration& cfg) {
```

Comment:
Why do we need this function in addition to `detail::load_plugins()`?

Reply:
Accidental duplication, sorry about that. Removed it.